pkg/components/metallb/component.go

```diff
@@ -78,7 +81,7 @@ func (c *component) RenderManifests() (map[string]string, error) {
 
 	t, err := util.RenderTolerations(c.SpeakerTolerations)
 	if err != nil {
-		return nil, errors.Wrap(err, "failed to marshal speaker tolerations")
+		return nil, fmt.Errorf("marshalling speaker tolerations: %w", err)
```

Comment:
Both "marshaling" and "marshalling" seem to be valid spellings. Go uses the former in the standard library. We currently have only one occurrence: https://github.com/kinvolk/lokomotive/blob/a175b3736d68954fbd4f9fc18c5f19accea62ac7/test/calico/calico_test.go#L72

I'm fine with either as long as we're consistent. "Marshalling" seems to be in wider use historically: https://books.google.com/ngrams/graph?smoothing=3&corpus=26&year_end=2019&content=marshalling%2Cmarshaling&year_start=1800&direct_url=t1%3B%2Cmarshalling%3B%2Cc0%3B.t1%3B%2Cmarshaling%3B%2Cc0

Reply:
I'll go with just `marshaling`, as we have more occurrences and our vendor directory also have more:
```
0 ✓ (637ms) 14:07:55 invidian@dellxps15mateusz ~/repos/kinvolk/lokomotive (invidian/errors-wrap)*$ git grep marshaling vendor/ | wc -l
138
0 ✓ (76.3ms) 14:07:56 invidian@dellxps15mateusz ~/repos/kinvolk/lokomotive (invidian/errors-wrap)*$ git grep marshalling vendor/ | wc -l
48
```